src/app/containers/StakePage/index.tsx

```diff
@@ -477,14 +483,22 @@ const InnerStakePage: React.FC = () => {
                     />
                   );
                 })}
-                <FeeBlock
-                  updateUsdTotal={updateUsdTotal}
-                  contractToken={AssetsDictionary.get(Asset.SOV)}
-                  title={t(translations.stake.vestingFees)}
-                  useNewContract
-                  frozen={frozen}
-                  vestedFees={true}
-                />
+                {vestingLoading === false &&
+                  vestingItems.length > 0 &&
+                  vestingItems.map(item => {
+                    if (item.type === 'origin') {
```

Comment:
I am not gonna force you to update it but the correct approach would be to change `VestGroup` to an `enum` (and rename it) because this is a magic string. But since it's used in many places and it would just add more complexity to this PR, leave it as is.

Reply:
Completely agree, I started down this route and realized how much additional refactoring would be required. Marking as resolved